[DOCU-1567] issue template updates (#2990)
Kong/docs.konghq.com: .github/ISSUE_TEMPLATE.md

<!-- 
Thank you for your interest in contributing!

Before submitting an issue, check our existing issues and pull requests to see if someone else has reported your issue previously. 

> If you know how to resolve the issue you're reporting, we highly encourage that you submit a pull request instead: https://help.github.com/articles/creating-a-pull-request/.

Reviewer: ```suggestion
If you know how to resolve the issue you're reporting, we encourage you to submit a pull request instead: https://help.github.com/articles/creating-a-pull-request/.
```

Thinking that if we want to discourage adverbs (in style guide) we could lead by example 😀

> If you have a troubleshooting issue, reach out to our community: https://github.com/Kong/kong/discussions.

Reviewer 2: Not sure how to phrase this, but I think this sentence needs to be a bit clearer - what exactly is a "troubleshooting issue"? Maybe make a point of drawing a contrast between code/product/etc troubleshooting vs something wrong with docs. 

E.g.: "If you need help troubleshooting issues with (your implementation? code? I don't know what phrase/word to use here), reach out to our community"

> If you have a troubleshooting issue, reach out to our community: https://github.com/Kong/kong/discussions.

Author: "If you have a code-related troubleshooting issue, please reach out to our community." Thoughts?

> If you have a code-related troubleshooting issue, please reach out to our community.: https://github.com/Kong/kong/discussions.

Reviewer 2: ```suggestion
If you have a code-related troubleshooting issue, please reach out to our community: https://github.com/Kong/kong/discussions.
```

-->

### What part of the site needs attention?

<!-- Link the part of the site that needs attention.  -->

### Summary

<!-- 
    Describe the issue in detail and, if applicable, tell us why this change is important.  

>         * Bad: "The ACL plugin documentation needs updating."
>         * Good: "Add the missing DELETE endpoint to ACL plugin documentation."

Reviewer: ```suggestion
        * Do: "Add the missing DELETE endpoint to ACL plugin documentation."
        * Don't: "The ACL plugin documentation needs updating."
```

Following the pattern we've got in the style guide already, plus it sounds more encouraging.
